### PR TITLE
kommander: fix bad wget | tar snippets

### PR DESCRIPTION
**Pull request by dkoshkin**
## Description of changes being made
Sorry I missed this in the original example since we didn't have anything to test with:
```
➜ wget "https://downloads.d2iq.com/dkp/v2.2.0/dkp-kommander-charts-bundle-v2.2.0.tar.gz" -O - | tar -xvf -

--2022-05-23 12:22:08--  https://downloads.d2iq.com/dkp/v2.2.0/dkp-kommander-charts-bundle-v2.2.0.tar.gz
Resolving downloads.d2iq.com (downloads.d2iq.com)... 108.138.94.98, 108.138.94.65, 108.138.94.103, ...
Connecting to downloads.d2iq.com (downloads.d2iq.com)|108.138.94.98|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1423360 (1.4M) [application/x-tar]
Saving to: ‘STDOUT’

-                                                                 0%[                                                                                                                                                     ]       0  --.-KB/s               x NOTICES.txt
-                                                               100%[====================================================================================================================================================>]   1.36M  3.56MB/s    in 0.4s

2022-05-23 12:22:09 (3.56 MB/s) - written to stdout [1423360/1423360]
➜  komm ll
total 1.4M
-rw-r--r-- 1 dkoshkin   72 May 21 08:10 NOTICES.txt
-rw-r--r-- 1 dkoshkin 1.4M Apr  7 22:36 dkp-kommander-charts-bundle-v2.2.0.tar.gz
```

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4488.s3-website-us-west-2.amazonaws.com/

## Checklist

- [x] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by mmcelroy91 — APPROVED** [review]

**Review by cmastr — APPROVED** [review]

File: pages/dkp/kommander/2.0/install/air-gapped/index.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ export VERSION=v2.0.0
 1.  Download the `kommander_image_bundle_v2.0.0_linux_amd64.tar` file.
 
     ```bash
-    wget "https://downloads.d2iq.com/kommander/airgapped/${VERSION}/kommander_image_bundle_${VERSION}_linux_amd64.tar" | tar -xvf -
+    wget "https://downloads.d2iq.com/kommander/airgapped/${VERSION}/kommander_image_bundle_${VERSION}_linux_amd64.tar" -O - | tar -xvf -
```
Comment (msdolbey) [inline pages/dkp/kommander/2.0/install/air-gapped/index.md:39]: I tried this - and it doesn't work. We should make sure we have the right image bundle for this - is this actually available?

```suggestion
    wget "https://downloads.d2iq.com/kommander/airgapped/${VERSION}/kommander_image_bundle_${VERSION}_linux_amd64.tar" -O - | tar -xvf -
```

Comment (msdolbey) [reply pages/dkp/kommander/2.0/install/air-gapped/index.md:39]: Is airgap actually available like this @takirala @karishmagadiya ?

**Review by msdolbey — APPROVED** [review]
👍 
So, everything looks good except for kommander 2.0 - but that could be sorted out in a new PR.
wget is short for we get it

**Comment by msdolbey** [comment]
well ... seeing as this is stuck in building ... I might just take care of this super quick

**Comment by mwrightd2** [comment]
> well ... seeing as this is stuck in building ... I might just take care of this super quick

Thanks @msdolbey !

**Comment by msdolbey** [comment]
I believe the linting would fail because anytime you delete a page it goes a bit berserk.

**Pull request merged**